File: FTP Client for WinForms using Ultimate FTP library/[VB]-FTP Client for WinForms using Ultimate FTP library/VB/MoveToRemoteFolder.vb
Namespace ClientSample
	Partial Public Class MoveToRemoteFolder
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Sub New(ByVal remoteDir As String)
			InitializeComponent()

			txtRemoteDir.Text = remoteDir
		End Sub

		''' <summary>
		''' Gets the path to the desired destination remote folder.
		''' </summary>
		Public ReadOnly Property RemoteDir() As String
			Get
				Return txtRemoteDir.Text
			End Get
		End Property

		''' <summary>
		''' Gets the search file masks.
		''' </summary>
		Public ReadOnly Property FileMasks() As String
			Get
				Return txtMasks.Text
			End Get
		End Property
	End Class
End Namespace

File: How to write an SFTP file transfer client for WinForms/[C#]-How to write an SFTP file transfer client for WinForms/C#/MoveToRemoteFolder.cs
using System.Windows.Forms;

namespace ClientSample
{
    public partial class MoveToRemoteFolder : Form
    {
        public MoveToRemoteFolder()
        {
            InitializeComponent();
        }

        public MoveToRemoteFolder(string remoteDir)
        {
            InitializeComponent();

            txtRemoteDir.Text = remoteDir;
        }

        /// <summary>
        /// Gets the path to the desired destination remote folder.
        /// </summary>
        public string RemoteDir
        {
            get { return txtRemoteDir.Text; }
        }

        /// <summary>
        /// Gets the search file masks.
        /// </summary>
        public string FileMasks
        {
            get { return txtMasks.Text; }
        }
    }
}